File: easy/python/issue19.py
#!/usr/bin/python2

"""
Create a function that takes a number as its only 
argument and returns True if it's less than or equal 
to zero, otherwise return False.

"""

def less_or_equal(num):
    if num <= 0:
        return True
    else:
        return False 

print(less_or_equal(10))
print(less_or_equal(0))

File: easy/c++/issue19.cpp
/* 
Create a function that takes an integer 
and returns true if it's divisible by 100, 
otherwise return false.
*/

#include <iostream>

using namespace std; 

bool divisible(int num) {

    if(num % 5 == 0 && num % 10 ==0) {
        return true; 
    }else{
        return false; 
    }
}

int main() {
    bool value = divisible(30);
    if(value == 1) {
        cout << "True" << endl; 
    }else{
        cout << "False" << endl; 
    }
    

return 0; 
}